tape/src/test/java/com/squareup/tape2/QueueFileLoadingTest.java

```diff
@@ -33,20 +33,32 @@
     assertTrue(testFile.delete());
     assertFalse(testFile.exists());
     QueueFile queue = new QueueFile.Builder(testFile).build();
-    assertEquals(0, queue.size());
-    assertTrue(testFile.exists());
+    try {
+      assertEquals(0, queue.size());
+      assertTrue(testFile.exists());
+    } finally {
+      queue.close();
```

Comment:
do most of these need the try-finally blocks (instead of only closing afterwards)? if there's an exception in these tests, does it matter if the file is closed?

Reply:
The problem is on Windows, the file won't be deleted if an exception occurs.

Reply:
Meaning they won't be cleaned up after the test. If that isn't a concern, I can remove the try-finally.

Reply:
oh, for my own understanding, do you mean that the `cleanup()` that deletes the file will fail if this isn't closed? if so, why is that?

Reply:
Yes, it fails because the file still has an open handle. Windows reports the file as being open which causes the delete to fail (Linux does not prevent deletion).

Reply:
Oh, that's interesting. Thanks for explaining it to me!

Reply:
i guess the close could in `cleanup()`? dunno which is preferable.

Reply:
That would work if QueueFile became a class member. It would still require explicit closing for cases that utilize multiple queues (QueueFileTest.testIterator, QueueTestFile.testAddAndRemoveElements, etc.).

Not sure the preference at this point. Maybe the easiest thing to do would just be to add the close() calls on those particular cases where it was failing on Windows. The try-finally could be removed as it isn't being applied on all cases consistently. Thus, there would be only be minor changes. At some point in the future a refactor could be done (try-with-resource might make the most sense).